src/electrum.rs

```diff
@@ -344,6 +344,18 @@ impl Rpc {
         }
     }
 
+    fn features(&self) -> Result<Value> {
+        Ok(json!({
+            "genesis_hash": self.tracker.chain().get_block_hash(0),
+            "hosts": {},
```

Comment:
Should we add `"localhost": { "tcp_port": electrum_rpc_port }`?

Reply:
:+1: